mods/creative/init.lua

```diff
@@ -119,9 +120,9 @@ creative.set_creative_formspec = function(player, start_i, pagenum, tab_id)
 		tooltip[creative_clear;Reset]
 		listring[current_player;main]
 		]] ..
-		"field[0.3,3.5;2.2,1;creative_filter;;".. filter .."]"..
+		"field[0.3,3.5;2.2,1;creative_filter;;".. inv.filter .."]"..
 		"listring[detached:creative_".. player_name ..";main]"..
-		"tabheader[0,0;creative_tabs;Crafting,All,Nodes,Tools,Items;".. tostring(tab_id) ..";true;false]"..
+		"tabheader[0,0;creative_tabs;Crafting,All,Nodes,Tools,Items;".. tostring(inv.tab_id) ..";true;false]"..
 		"list[detached:creative_".. player_name ..";main;0,0;8,3;".. tostring(start_i) .."]"..
 		"table[6.05,3.35;1.15,0.5;pagenum;#FFFF00,".. tostring(pagenum) ..",#FFFFFF,/ ".. tostring(pagemax) .."]"..
 		default.get_hotbar_bg(0,4.7)..
```

Comment:
It was intentional since these lines were quite long. Also it's not a requirement of the [Lua guidelines](http://dev.minetest.net/Lua_code_style_guidelines).

> Spaces are used around binary operators. With the exception of the member access operator (".") where there shouldn't be spaces, and **the concatenation operator ("..") where spaces are optional**.

But I will change it.